pkg/controller/associated/android.go

```diff
@@ -39,7 +39,7 @@ func (c *Controller) getAndroidData(region string) ([]AndroidData, error) {
 		return nil, fmt.Errorf("unable to lookup realm: %w", err)
 	}
 
-	apps, err := c.db.ListActiveAppsByOS(realm.ID, database.OSTypeAndroid)
+	apps, err := c.db.ListActiveApps(realm.ID, []database.Scope{database.WithAppOS(database.OSTypeAndroid)}...)
```

Comment:
Sorry to be a butt, but I don't think we need to allocate a slice here. The signature is variadic, so you should be able to:

```go
apps, err := c.db.ListActiveApps(realm.ID, database.WithAppOS(database.OSTypeAndroid))
```

Reply:
oops... I was not paying attention